experimental/SetPartitions/src/PartitionProperties.jl

```diff
@@ -0,0 +1,254 @@
+"""
+size(p::Partition)
+
+This function outputs the size of the input partition (i.e. the number of points in `p`).
+
+# Arguments
+- `p`: Input partition
+
+# Returns
+- size of `p`
+"""
+function size(p::Partition)
+    length(p.lower_points) + length(p.upper_points)
+end
+
+function is_composable(p::Partition, q::Partition)
+    length(p.upper_points) == length(q.lower_points)
+end
+
+function get_upper_points(p::Partition)
+    p.upper_points
+end
+
+function get_lower_points(p::Partition)
+    p.lower_points
+end
+
+"""
+size(p::ColoredPartition)
+
+This function outputs the size of the input colroed partition (i.e. the number of points in `p`).
+
+# Arguments
+- `p`: Input colored partition
+
+# Returns
+- size of `p`
+"""
+function size(p::ColoredPartition)
+    size(p.partition)
+end
+
+function is_composable(p::ColoredPartition, q::ColoredPartition)
+    p.color_upper_points == q.color_lower_points && is_composable(p.partition, q.partition)
+end
+
+function get_upper_points(p::ColoredPartition)
+    p.partition.upper_points
+end
+
+function get_lower_points(p::ColoredPartition)
+    p.partition.lower_points
+end
```

Comment:
Better names (better in the sense of consistency with the rest of Oscar) would be `upper_points` and `lower_points`.